docs/content/docs/dev/table/sql/insert.md

```diff
@@ -208,7 +208,9 @@ column_list:
 **COLUMN LIST**
 
 Given a table T(a INT, b INT, c INT), Flink supports INSERT INTO T(c, b) SELECT x, y FROM S. The expectation is
-that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable). 
+that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable).
+For the connector developers who want to avoid overwriting non-target columns with null values when processing partial column updates,
```

Comment:
Also, I'm wondering whether it's more clear to break the line in here since these lines are more likely to say other things.

Reply:
current version: ![image](https://github.com/apache/flink/assets/3712895/601cd172-3d3f-4ea8-8d64-c40b360908b2)
a new line seems to be better, will update it.